ouroboros-consensus-shelley-test/test/Test/ThreadNet/RealTPraos.hs

```diff
@@ -45,47 +54,85 @@ import           Test.ThreadNet.Util.Seed (runGen)
 
 type Crypto = TPraosMockCrypto ShortHash
 
+-- | A number in @[0,10]@.
+newtype Tenths = Tenths Word64
```

Comment:
When I read `Tenths`, I think about `1/10, 2/10, ..., 10/10`.

Reply:
"Tenth" behaves like a unit; compare to `newtype Seconds = Seconds Word64`, for example.

Does that observation influence your preference?

Reply:
> Does that observation influence your preference?

Sorry, no :grinning:

Reply:
No sweat off my back :) I will change it.